app/controllers/bulkrax/importers_controller.rb

```diff
@@ -35,7 +35,10 @@ def show
       elsif defined?(::Hyrax)
         add_importer_breadcrumbs
         add_breadcrumb @importer.name
-
+        @work_entries = @importer.entries.where(type: @importer.parser.entry_class.to_s).page(params[:work_entries_page]).per(30)
```

Comment:
what am I missing? it really looks like the elseif and else are identical here. wouldn't it be better to just put an `end` on line 37?